rel/expr.go

```diff
@@ -86,3 +87,29 @@ func EvalExpr(ctx context.Context, expr Expr, local Scope) (_ Value, err error)
 	//TODO: this is only the initial scope, how to get the last scope?
 	return expr.Eval(ctx, local)
 }
+
+// Errors is sequence of error
+type Errors struct {
+	errors []error
+}
+
+// Error returns string represents Errors
+func (errs Errors) Error() string {
+	if len(errs.errors) == 0 {
+		return ""
+	}
+
+	var result strings.Builder
+	for i, err := range errs.errors {
+		if i > 0 {
+			result.WriteString("\n\n")
+		}
+		result.WriteString(err.Error())
+	}
+	return result.String()
+}
+
+// CmdIdentifier identifies an arrai command like test, shell and eval etc.
+type CmdIdentifier struct {
+	Name string
+}
```

Comment:
This should live in `internal/test/`, since it should only be used in the case of testing.

In general, test-related code should not be mixed in with implementation code.